adapters/Unity/UnityAdapter/GADMUnityInterstitialAd.m

```diff
@@ -118,18 +118,18 @@ - (void)presentInterstitialFromRootViewController:(UIViewController *)rootViewCo
     return;
   }
 
-  if (!_placementID) {
+  if (!_loadedPlacementID) {
     NSLog(@"Unity Ads received call to show before successfully loading an ad");
   }
   
   [strongConnector adapterWillPresentInterstitial:strongAdapter];
```

Comment:
Since your SDK is now responsible for sending "showFailed" callbacks, please move the equivalent "willPresent" callbacks into the unityAdsDidStart event. Please apply for interstitial and rewarded.

If the adapter sends back a "willPresent" callback before a "didFailToPresent" callback, the "didFailToPresent" never makes it back to the app, as "willPresent" is assumed to mean a successful present.